tests/test_models/test_backbones/test_timm_backbone.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import pytest
+import torch
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmcls.models.backbones import TIMMBackbone
+
+
+def check_norm_state(modules, train_state):
+    """Check if norm layer is in correct train state."""
+    for mod in modules:
+        if isinstance(mod, _BatchNorm):
+            if mod.training != train_state:
+                return False
+    return True
+
+
+def test_timm_backbone():
```

Comment:
I think we can use this method to skip the unit test. (We shouldn't modify some specific workflows to ignore it, because that may cause our workflows to fail if `timm` makes some other changes in the future.)

```suggestion
from mmcls.models.backbones.timm_backbone import timm

@pytest.mark.skipif(timm is None, reason='timm is not available.')
def test_timm_backbone():
```